packages/pn-personafisica-webapp/src/pages/NuovaDelega.page.tsx

```diff
@@ -153,6 +150,28 @@ const NuovaDelega = () => {
       setLoadAllEntities(true);
     }
   };
+  // done for complexity
```

Comment:
What this comment refers for?

Reply:
need delete it? I commented because i write function of autocomplete out of the component